LBH.AdultSocialCare.Api/V1/Infrastructure/DatabaseContext.cs

```diff
@@ -34,7 +33,7 @@ public DatabaseContext(DbContextOptions options)
         public DbSet<HomeCarePackageSlots> HomeCarePackageSlots { get; set; }
         public DbSet<Users> Users { get; set; }
         public DbSet<Clients> Clients { get; set; }
-        public DbSet<Status> Status { get; set; }
+        public DbSet<PackageStatus> Status { get; set; }
```

Comment:
The dbset/table can be named PackageStatus as well. Would make more sense for someone working on the DB

Reply:
Thanks Duncan